src/main/resources/io/viash/platforms/nextflow/VDSL3Helper.nf

```diff
@@ -950,13 +958,10 @@ def workflowFactory(Map args) {
 
         def out = [ output[0], outputFiles ]
 
-        // passthrough additional items
-        if (output[1]) {
-          out.addAll(output[1])
-        }
-
         out
```

Comment:
Might as well simplify to `[  output[0], outputFiles ]`

```suggestion
        [  output[0], outputFiles ]
```

Reply:
Indeed, adjusted!